kiwi/container/docker.py

```diff
@@ -28,9 +34,26 @@ class ContainerImageDocker(object):
 
     * :attr:`root_dir`
         root directory path name
+
+    * :attr:`custom_args`
+        custom argument hash:
+        - container_name: tag name of the container, default: latest
+        - entry_command: default command to run, default: bin/bash
     """
-    def __init__(self, root_dir):
+    def __init__(self, root_dir, custom_args=None):
         self.root_dir = root_dir
+        self.docker_dir = None
+        self.docker_root_dir = None
+
+        if custom_args and 'container_name' in custom_args:
+            self.container_name = custom_args['container_name']
+        else:
+            self.container_name = 'latest'
+
+        if custom_args and 'entry_command' in custom_args:
+            self.entry_command = custom_args['entry_command']
```

Comment:
note that entry_command may not be the best variable name since umoci supports `entry_point` and `cmd` configuration tags. I would go for direct mapping, lets say 'self.entry_point' and `self.cmd`.